workspace-manager-clienttests/README.md

```diff
@@ -49,6 +49,17 @@ Perf Test
 ```
 ./gradlew  runTest --args="suites/BasicPerf.json /tmp/TR"
 ```
+#### Upload test results to Google Bucket
+To upload Test Runner results to a Google Bucket.
+Specify the location of the bucket in the `uploadlists` directory.
+For example, the `CompressDirectoryToBucket.json` specifies the 
+name of the bucket and the Java class in the `uploadscripts` directory for uploading the test results.
```

Comment:
Nit: The bucket location is specified as a parameter in an upload config file located in the resources/uploadlists directory.

Reply:
Updated comments.